tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -968,6 +1004,11 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
  private:
   /// Maintain info about nodes to rewrite
   std::vector<RewriteInfo> rinfo_;
+  /// Mantain info about nodes to rewrite with additional
```

Comment:
nit: spelling `Maintain`, and all comments are supposed to be sentences with punctuation (i.e. trailing period).

Reply:
Thanks for the comment. Fixed in [59cc901](https://github.com/tensorflow/tensorflow/commit/59cc9014114ac5416e8980759ac6d105b9ae116d) and also all comments now have trailing period.